examples/getting_started/1-Introduction.ipynb

```diff
@@ -44,7 +44,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This is the standard way to make the numpy and pandas libraries available in the namespace. We recommend always importing HoloViews as ``hv`` and if you haven't already installed HoloViews, check out our [installation page].\n",
+    "This is the standard way to make the numpy and pandas libraries available in the namespace. We recommend always importing HoloViews as ``hv`` and if you haven't already installed HoloViews, check out the install instructions on our [homepage](www.holoviews.org).\n",
```

Comment:
It seems there is bad link to homepage.
The right one should be http://www.holoviews.org

Reply:
Thanks!